src/server/ftpserver.rs

```diff
@@ -467,6 +470,27 @@ where
         self
     }
 
+    /// Enables a password guessing protection policy
+    ///
+    /// Policy used to temporarily lock an account, source IP or the
+    /// combination of both, after a certain number of failed login
+    /// attempts for a certain time.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use libunftp::Server;
+    /// use libunftp::options::FailedLoginsPolicy;
+    /// use unftp_sbe_fs::ServerExt;
+    ///
+    /// // Use it in a builder-like pattern:
+    /// let mut server = Server::with_fs("/tmp").failedloginspolicy(FailedLoginsPolicy::SourceUserLock(FailedLoginsPenalty::new()));
+    /// ```
+    pub fn failedloginspolicy(mut self, failedloginspolicy_option: FailedLoginsPolicy) -> Self {
```

Comment:
If we don't have Off as an option too but Optional for that then perhaps the function should take `failedloginspolicy_option :Option<FailedLoginsPolicy>`. This will allow users to also switch it off explicitly (I know not calling this method does the same, but then its not explicit).

Method name should be `failed_logins_policy` I think. The parameter can probably be just called `policy`

Reply:
Renamed. Yes my intention was that the 'failed logins' cache was an option defaulting to None. I have pushed a new commit to make this the default.